api/tests/routes/pro/signup_pro_test.py

```diff
@@ -376,3 +376,16 @@ def when_invalid_phone_number(self, app):
         assert response.status_code == 400
         error = response.json
         assert "phoneNumber" in error
+
+    def test_when_invalid_phone_number(self, app):
+        # Given
+        data = BASE_DATA_PRO.copy()
+        data["phoneNumber"] = "0873492896"
+
+        # When
+        response = TestClient(app.test_client()).post("/users/signup/pro", json=data)
```

Comment:
(détail) Vivons dans le présent et utilisons la fixture `client` :

```suggestion
    def test_when_invalid_phone_number(self, client):
        # Given
        data = BASE_DATA_PRO.copy()
        data["phoneNumber"] = "0873492896"

        # When
        response = client.post("/users/signup/pro", json=data)
```

idem dans les autres tests ;)

Reply:
yep. j'avoue avoir copié/collé sans me posé trop de question :)
et tant qu'a vivre dans le présent, je vais aussi utiliser `url_for` au lieu de ces vilaines routes en dur